### DISCUSSION TITLE
Pull request badge is not aligned with other badges

### DISCUSSION THREAD

**Discussion by romgille** [community/community#48473] [Projects and Issues]
### Select Topic Area

Bug

### Body

Hi, when navigating the kanban view, pull request badges are not aligned with other badges.

![image](https://user-images.githubusercontent.com/12200233/221138991-c920ddd7-4f21-4f71-80e5-5655e341778e.png)

Does anyone see a similar issue ?

Thank you in advance !

**Comment by nul800sebastiaan** [comment]
Yes, seeing this too: 

![image](https://user-images.githubusercontent.com/304656/221150200-3b886304-2cf6-4135-9cc2-92f339b4bf07.png)

**Comment by olavis** [comment]
<img width="344" alt="Screenshot 2023-02-23 at 09 13 34" src="https://user-images.githubusercontent.com/38139899/221798646-b470ccf0-7928-4155-9280-43c3e2934d3e.png">
Yup

**Comment by iansan5653** [comment]
Hello, thanks for the report! We are working on a fix and will roll it out within the next day.

**Reply by romgille** [reply]
Thanks ! 💪🏻
